sample/src/Main.fs

```diff
@@ -1,30 +1,43 @@
 ﻿module Main
 
+open Browser.Types
 open Lit
 open Fable.Haunted
 open Fable.Haunted.Types
 
+type EventTarget with
+    member this.Value = (this :?> HTMLInputElement).value
```

Comment:
There's a bunch of event related functions that aren't particularly tied to Haunted (you could technically use them in Sutil too) I think I might move them in a separate package eventually this one looks like something I missed in other codes